index.js

```diff
@@ -126,13 +126,9 @@ cm.Item({
 });
 
 function checkLocation(opts) {
```

Comment:
So, with checkLocation function (without checkLocation, I couldn't get these to work): 
<img width="882" alt="screen shot 2016-06-27 at 1 14 51 pm" src="https://cloud.githubusercontent.com/assets/9204835/16388880/3665f9be-3c69-11e6-88d3-cc599b2bd955.png">
<img width="878" alt="screen shot 2016-06-27 at 1 14 05 pm" src="https://cloud.githubusercontent.com/assets/9204835/16388884/3ce3fde0-3c69-11e6-8eb9-657bd95fdd39.png">